src/components/Image/Image.st.css

```diff
@@ -1,10 +1,61 @@
+:import {
+  -st-from: "../../common/formatters.st";
+  -st-named: color, fallback, applyUnit;
+}
+
+/*Defaults*/
+:vars {
+  defaultBorderWidth: 0px;
+  defaultBorderRadius: 0px;
+  defaultContainerOpacity: 100%;
+  defaultImageOpacity: 100%;
+}
+
+/* Overrides */
+:vars {
+  /*
+    A background color by a color from the palette for the container
+  */
+  ContainerBackgroundColor: --overridable;
+  /*
+    A border color by a color from the palette for the container
+  */
+  ContainerBorderColor: --overridable;
+  /*
+    A border width in pixels for the container
+    @default 0px
+  */
+  ContainerBorderWidth: --overridable;
+  /*
+    A border radius in pixels for the container
+    @default 0px
+  */
+  ContainerBorderRadius: --overridable;
+  /*
+    An opacity between 0 to 100 for the container
+    @default 100%
+  */
+  ContainerOpacity: --overridable;
+  /*
+    An opacity between 0 to 100 for the image itself
+    @default 100%
+  */
+  ImageOpacity: --overridable;
+}
+
 :vars {
-  BlurRadius: 3px;
+  backgroundColor: color(value(ContainerBackgroundColor));
+  borderColor: color(value(ContainerBorderColor));
```

Comment:
I think we should have fallback colors here as well:)

Reply:
We can set a default color for the border (because the default width is 0px), but in case of the image it will appear by default if the image doesn't fully cover the container - not sure if we want this case?

Reply:
So since you said that it works fine now you don't need to change anything:)